src/conf.c

```diff
@@ -236,7 +236,15 @@ conf_init(conf_t * conf)
 
 	if ((s2 = getenv("HOME")) != NULL) {
 		char s[MAX_STRING];
-		sprintf(s, "%s/%s", s2, ".axelrc");
+
+		/* ensure that s2 can fit in s */
+		if (strlen(s2) > (sizeof(s) - strlen("/.axelrc") - 1))
```

Comment:
snprintf can be used instead and the return value checked

Reply:
yeah that too. I thought this was cleaner to read :) but I can switch to snprintf

Reply:
Actually, are you sure I can use snprintf and achieve the same result? snprintf will return the number of matched objects. If the string was longer than the maximum length it will still be reported as "matched", thus it won't be possible to distinguish a case of string too long.